open_academy/models/open_academy_session.py

```diff
@@ -1,14 +1,52 @@
-from odoo import models, fields
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
 
 
 class OpenAcademySession(models.Model):
     _name = "open.academy.session"
     _description = "model of session on open academy"
 
     name = fields.Char(required=True, size=50)
-    duration = fields.Text()
-    start_date = fields.Datetime()
-    number_seats = fields.Integer()
+    duration = fields.Integer()
+    active = fields.Boolean(default=True)
+    start_date = fields.Datetime(default=fields.Date.context_today)
+    end_date = fields.Datetime()
+    number_seats = fields.Integer(default=50, required=True)
+    number_seats_taken = fields.Integer()
     instructor_id = fields.Many2one("res.partner", "instructor")
     course_id = fields.Many2one("open.academy.course", required=True)
     attendees_ids = fields.Many2many("res.partner", "session_res_partner_rel")
+    available_seats_computed = fields.Integer(compute="percentage_computed")
```

Comment:
Why this field? The exercise says `Add the percentage of taken seats to the Session model`

So the computed field should be `number_seats_taken` and the maths for that is number of attendees / number of seats (validate the number of seats first).

Also, did you run precommit for this change? 
There's a lint for computed method's name, please run precommit and change this.
For good practice, the name of the method is placed with the name of the field

See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models) 

```suggestion
    available_seats_computed = fields.Integer(compute="_compute_percentage_available_seats")
```

Please do not put the word computed in the field name, remember that this name is used for the string that is displayed to the user.

```suggestion
    available_seats = fields.Integer(compute="percentage_computed")
```